lib/services/computeManagement2/package.json

```diff
@@ -12,7 +12,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Sun, Carissa <caris@microsoft.com>"
   ],
-  "version": "1.0.0-preview",
+  "version": "2.0.0-preview",
```

Comment:
The API version mostly reflects the service side change and promise, and I'm not sure about the client side requirement of removing the ```-preview``` suffix yet. So, I keep it as-is. Is it OK?